ingest/src/main/java/com/connexta/ingest/transform/TransformClient.java

```diff
@@ -6,53 +6,52 @@
  */
 package com.connexta.ingest.transform;
 
+import com.connexta.transformation.rest.models.TransformRequest;
+import com.connexta.transformation.rest.models.TransformResponse;
 import java.net.URI;
 import java.net.URISyntaxException;
+import javax.annotation.PostConstruct;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpEntity;
-import org.springframework.http.ResponseEntity;
+import org.springframework.http.HttpHeaders;
 import org.springframework.stereotype.Service;
-import org.springframework.web.client.ResponseErrorHandler;
 import org.springframework.web.client.RestTemplate;
 
 @Service
 public class TransformClient {
 
-  private final RestTemplate restTemplate;
-  private final URI transformEndpoint = new URI("TODO");
+  private static final Logger LOGGER = LoggerFactory.getLogger(TransformClient.class);
 
-  @Autowired
-  public TransformClient(RestTemplate restTemplate) throws URISyntaxException {
-    this.restTemplate = restTemplate;
+  @Autowired private RestTemplate restTemplate;
+
+  private URI transformEndpoint;
+
+  @PostConstruct
+  private void setEndpoint() throws URISyntaxException {
+    // TODO: Set endpoint URL
+    setTransformEndpoint("http://TODO");
```

Comment:
✏️ This endpoint can come from the configuration file

Reply:
>  This endpoint can come from the configuration file

Do you mean the application.yml file?

Reply:
Yeah it seems like you should inject this.

Reply:
We can address this in a future commit. See https://github.com/connexta/multi-int-store/pull/47.